### PR TITLE
Added 08/04 Upcoming Releases Update

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -8,33 +8,35 @@ The Teleport team delivers a new major release roughly every 3 months.
...
 
-| Version     | Date       | Description                |
-|-------------|------------|----------------------------|
-| Release     | July 2022  | Good to go for production. |
+- support for directory sharing
+- simplified configuration of AD
+- configurable playback speed for recorded sessions
 
-### Teleport 10.2
+Server access will add:
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:19]:
```suggestion
Server Access will add:
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -8,33 +8,35 @@ The Teleport team delivers a new major release roughly every 3 months.
...
+- support for Auditd
+- support for non-disk based certificates
+- automatic discovery of servers on AWS
+
+For Teleport Connect, we will Touch ID support in macOS builds, and introduce
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:24]:
```suggestion
For Teleport Connect, we will add Touch ID support in macOS builds, and introduce
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -8,33 +8,35 @@ The Teleport team delivers a new major release roughly every 3 months.
...
 
 | Version     | Date         | Description                |
 |-------------|--------------|----------------------------|
 | Release     | August 2022  | Good to go for production. |
 
 ### Teleport 10.3
 
-Teleport 10 will bring Cassandra support to Database Access.
+Teleport 10 will bring Cassandra support to Database Access. Host
+certificate support will be added to Machine ID
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:39]:
```suggestion
certificate support will be added to Machine ID.
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**